docs/operator-manual/applicationset/Generators-Merge.md

```diff
@@ -113,6 +122,57 @@ When merged with the updated base parameters, the `values.redis` value for the p
   values.redis: 'true'
 ```
 
+## Merge Modes (Join types)
+With the `uniq` merge modes (`left-join-uniq`, `inner-join-uniq`, `full-join-uniq`), all the parameter sets produced by any of the generators under a Merge generator, need to have unique values for the configured merge keys.
+
+However, this behaviour can be changed by using any of the merge modes that do not end with `-uniq`. This will allow more than one parameter sets of (only) the **base generator** to have the same values for the merge keys.
```

Comment:
would it be helpful to provide the equivalent SQL to explain joins? It seems like the `-uniq` is kind of like using `DISTINCT`, but rather, instead of actually doing distinct, it just errors if there are merge keys that are identical.

Reply:
We cannot expect users to be familiar with SQL. The documentation should be simple enough for a user not familiar with left/inner/full terminology to understand the behavior of these modes.

Something like
```
With the `uniq` merge modes (`left-join-uniq`, `inner-join-uniq`, `full-join-uniq`), all the parameter sets produced by any generators under a Merge generator need to have unique values for the configured merge keys. An error preventing the generation will be returned when duplicated keys are found.
```

However, I am still not sure what you are trying to explain with the last sentence of the second paragraph.